thoth/package_extract/image.py

```diff
@@ -502,4 +503,5 @@ def run_analyzers(path: str, timeout: int = None) -> dict:
         "python-files": _gather_python_file_digests(path),
         "operating-system": _gather_os_info(path),
         "system-symbols": _get_system_symbols(path),
+        "python_version": platform.python_version(),
```

Comment:
This is python version under which the package-extract runs (BTW it's stated in metadata of the JSON document produced). You will need to check what is present in in extract container image (`${path}/usr/bin/python*`).